src/subscribables/subscribable.js

```diff
@@ -150,8 +168,10 @@ var ko_subscribable_fn = {
 };
 
 ko.exportProperty(ko_subscribable_fn, 'subscribe', ko_subscribable_fn.subscribe);
+ko.exportProperty(ko_subscribable_fn, 'subscribeAll', ko_subscribable_fn.subscribeAll);
 ko.exportProperty(ko_subscribable_fn, 'extend', ko_subscribable_fn.extend);
 ko.exportProperty(ko_subscribable_fn, 'getSubscriptionsCount', ko_subscribable_fn.getSubscriptionsCount);
+ko.exportProperty(ko_subscribable_fn, 'allEvent', allEvent);
```

Comment:
Actually exporting it is useful. If you need to programmatically subscribe to events you may want to subscribe directly to the all event type. It's a covered test case as part of this pull request, actually. SubscribeAll is just a convenience method.